ramls/order_invoice_relationship.raml

```diff
@@ -0,0 +1,48 @@
+#%RAML 1.0
+title: "Orders Storage"
+baseUri: http://github.com/folio-org/mod-orders-storage
+version: v3
+
+documentation:
+  - title: Order relationship
+    content: <b>CRUD APIs used to manage relationship between order and invoice.</b>
+
+types:
+    order-invoice-relationship: !include acq-models/mod-orders-storage/schemas/order_invoice_relationship.json
+    order-invoice-relationship-collection: !include acq-models/mod-orders-storage/schemas/order_invoice_relationship_collection.json
+    UUID:
+     type: string
+     pattern: ^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[1-5][0-9a-fA-F]{3}-[89abAB][0-9a-fA-F]{3}-[0-9a-fA-F]{12}$
+
+traits:
+    orderable: !include raml-util/traits/orderable.raml
+    pageable:  !include raml-util/traits/pageable.raml
+    searchable: !include raml-util/traits/searchable.raml
+    language: !include raml-util/traits/language.raml
+
+resourceTypes:
+    collection: !include raml-util/rtypes/collection.raml
+    collection-item: !include raml-util/rtypes/item-collection.raml
+
+/order-storage/order-invoice-relns:
+  type:
+    collection:
+      exampleCollection: !include acq-models/mod-orders-storage/examples/order_invoice_relationship_collection.sample
+      exampleItem: !include acq-models/mod-orders-storage/examples/order_invoice_relationship_post.sample
+      schemaCollection: order-invoice-relationship-collection
+      schemaItem: order-invoice-relationship
+  get:
+    description: Get list of relationships
+    is: [
+      searchable: {description: "with valid searchable fields: for example code", example: "[\"code\", \"MEDGRANT\", \"=\"]"},
```

Comment:
Please, use valid example and description of searchable trait.  `order_invoice_relationship` contains only two fields: `invoiceId` and `purchaseOrderId`, there is no `code` field.

Reply:
Done. Check again please.